apps/serialization/messages.proto

```diff
@@ -209,6 +213,9 @@ message Pool {
   uint64 owner_id = 1;
 }
 
+message Bush {
```

Comment:
Why do we need this empty message struct? is it for the addional_info oneof? 🤔

Reply:
yeah! exactly that